portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,55 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="title">
+      <h1>Welcome to Jason's Portfolio</h1>
     </div>
+
+    <div id="header">
+      <img class="circular--square" src="../Images/headshot.png" alt="My headshot">
+      <p> Jason Ah Chuen <br>
+          Stanford University | Class of 2022 <br>
+          B.S. Candidate | Computer Science <br>
+          jahchuen (at) stanford (dot) edu
+      </p>
+    </div>
+
+    <div id="content1">
```

Comment:
note that you have 2 divs with identical "content1" IDs (here and at line 50). It's good practice to make IDs unique, and meaningfully named. Can you think of better (and distinct) names for these divs?